subsys/bluetooth/Kconfig.logging

```diff
@@ -0,0 +1,1083 @@
+# Bluetooth logging configuration options
+
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BT_LOG
+        bool "Bluetooth logging"
+	default y if LOG && BT
+	imply LOG_FUNC_NAME_PREFIX_INF
+	imply LOG_FUNC_NAME_PREFIX_WRN
+	imply LOG_FUNC_NAME_PREFIX_ERR
+	imply LOG_FUNC_NAME_PREFIX_DBG
+	select BT_DEBUG
+	select BT_LOG_LEGACY
+
+if BT_LOG
+
+menuconfig BT_LOG_LEGACY
+	bool "Bluetooth legacy logging options"
+
+if BT_LOG_LEGACY
+
+# COMMON
+
+config BT_DEBUG_HCI_DRIVER
+	bool "[DEPRECATED] Bluetooth HCI driver debug"
+	select DEPRECATED
+	help
+	  This option enables debug support for the active
+	  Bluetooth HCI driver, including the Controller-side HCI layer
+	  when included in the build.
+
+config BT_DEBUG_RPA
+	bool "[DEPRECATED] Bluetooth Resolvable Private Address (RPA) debug"
+	select DEPRECATED
+	depends on BT_RPA
+	help
+	  This option enables debug support for the Bluetooth
+	  Resolvable Private Address (RPA) generation and resolution.
+
+menu "[DEPRECATED] Audio"
```

Comment:
I think we can just remove all the deprecated log options for LE Audio, as they are all experimental, and doesn't actually warrant a deprecation.

Reply:
That is one of our goal to remove the deprecated symbols. I will do it in a following PR if it's fine for you.